libdnf-cli/output/changelogs.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "libdnf-cli/output/changelogs.hpp"
+
+#include "libdnf-cli/tty.hpp"
+
+#include "libdnf/rpm/package_set.hpp"
+
+#include <ctime>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+
+namespace libdnf::cli::output {
+
+void print_changelogs(
```

Comment:
There are more things that IMHO could be refactored, but I understand this was only moved from different file.